addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -422,27 +422,27 @@
         t-att-data-quiz-karma-won="quiz_karma_won"
         t-att-data-has-next="1 if next_slide else 0"
         t-att-data-next-slide-url="'/slides/slide/%s' % (slug(next_slide)) if next_slide else None">
-        <div t-foreach="slide.question_ids" t-as="question"
-            class="o_wslides_js_lesson_quiz_question mt-3" t-att-data-question-id="question.id" t-att-data-title="question.question">
+        <div t-foreach="questions" t-as="question"
```

Comment:
Would it be possible to rename questions in the returned structure of _get_slide_quiz_data to slide_questions ? It would make code easier to understand (and grep). Maybe check in a separate commit to see the related diff of that change ...